test/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+file(GLOB test
+        "*.cpp"
+        )
```

Comment:
pre-existing but the indentation seems inconsistent here.

Reply:
Thanks, fixed.